1_fetch/src/data_extraction_ms.R

```diff
@@ -6,31 +6,29 @@
 ################################
 
 
-## Libs
-library(dplyr)
-library(readr)
-library(sbtools)
+## Libs - Commented out for targets
+# library(dplyr)
+# library(readr)
+# library(sbtools)
 
+## Check that you are in project home directory - should be in project home directory - Commented out for Targets 
+# print(getwd())
 
-## Check that you are in project home directory - should be in project home directory
-print(getwd())
-
-## Global Variables
-
-out_path <- '1_fetch/out'
-csv_filename_output <- 'model_RMSEs.csv'
-sb_id_sciencebase <- '5d925066e4b0c4f70d0d0599'
-
+## Global Variables - Commented out for targets
+# downloaded_data_folder <- '1_fetch/out'
+# file_name <- 'model_RMSEs.csv'
+# sb_id_sciencebase <- '5d925066e4b0c4f70d0d0599'
+ 
 
 ### FUNCTION 1 - download the right data from Sciencebase. This function returns the path to the downloaded data 
 
 data_extraction <- function(filename_output_csv,
-                            download_data_location,
+                            downloaded_data_location,
                             selected_id_sciencebase,
                             sb_filenames = 'me_RMSE.csv') {
   # print()
```

Comment:
Same thing with stray debug statements and such - now that the code's well structured and running nicely, we can clean it up and improve readability by removing these!